### PR TITLE
Add and option switch for installing cereal 

### PR DESCRIPTION
**Pull request by phlptp**
which defaults false everywhere but visual studio

### Description

If merged this pull request will

### Proposed changes

-

### CODE REVIEW

File: config/HELICSConfig.cmake.in [hunk 1]
```diff
@@ -124,7 +124,7 @@ find_program(HELICS_APP helics_app
 	)
 	
 find_program(HELICS_BROKER helics_broker 
-	PATHS ${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_BINDIR@
+	PATHS c
```
Comment (nightlark) [inline config/HELICSConfig.cmake.in]: Was this change intentional?

Comment (phlptp) [reply config/HELICSConfig.cmake.in]: no,  I was tinkering with that file before I saw your latest PR and I like yours better so that file should not have changed.  I probably screwed it up,  If you like the changes in the other file just add them to yours and I can delete the branch.
